src/Razor/src/Microsoft.VisualStudio.RazorExtension/source.extension.vsixmanifest

```diff
@@ -31,9 +31,6 @@
     <Asset Type="Microsoft.ServiceHub.Service" d:Source="File" Path="Microsoft.VisualStudio.Razor.TagHelperProvider64S.servicehub.service.json" />
     <Asset Type="Microsoft.ServiceHub.Service" d:Source="File" Path="ServiceHubCore\Microsoft.VisualStudio.Razor.TagHelperProviderCore64.servicehub.service.json" />
     <Asset Type="Microsoft.ServiceHub.Service" d:Source="File" Path="ServiceHubCore\Microsoft.VisualStudio.Razor.TagHelperProviderCore64S.servicehub.service.json" />
-    <Asset Type="Microsoft.VisualStudio.Assembly" Path="Microsoft.VisualStudio.LanguageServer.Protocol.dll" />
```

Comment:
@matteo-prosperi I didn't see you mention it in the mail and I'm pretty sure it doesn't actually matter at runtime but folks may also want to remove their VSIXManifest for completeness

Reply:
I don't have a recommendation at this time and we are explicitly not asking people to remove the dlls from their vsixes. But that will likely be done for 17.1